data/brands/shop/shoes.json

```
@@ -22,6 +22,18 @@
         "shop": "shoes"
       }
     },
+    {
+      "displayName": "SAS (San Antonio Shoemakers)",
+      "locationSet": {"include": ["001"]},
+      "matchNames": ["SAS Shoes", "SAS", "San Antonio Shoe","San Antonio Shoemakers"],
```

This is just a comment, and doesn't matter at all:  Because "SAS" is used as a value on the `name` tag, it doesn't need to be included in the `matchTags` array.  It's already matched by being a `name`.  When our build script runs, the redundant names will be removed automatically from this list.  Just letting you know for future contribution, or in case you notice the next commit remove it.

```suggestion
      "matchNames": ["SAS Shoes", "San Antonio Shoe", "San Antonio Shoemakers"],
```